lucky_draw/priv/repo/seeds.exs

```diff
@@ -1,11 +1,22 @@
-# Script for populating the database. You can run it as:
-#
-#     mix run priv/repo/seeds.exs
-#
-# Inside the script, you can read and write to any of your
-# repositories directly:
-#
-#     LuckyDraw.Repo.insert!(%LuckyDraw.SomeSchema{})
-#
-# We recommend using the bang functions (`insert!`, `update!`
-# and so on) as they will fail if something goes wrong.
+alias LuckyDraw.Repo
+alias LuckyDraw.Lottery.Award
+
+opts = [on_conflict: :replace_all, conflict_target: :id]
+
+Repo.insert!(%Award{id: 1, content: "The Last of Us II", price: 590}, opts)
+
+Repo.insert!(
+  %Award{
+    id: 2,
+    content: "PS5 Limited Edition The Last of Us II GOTY Bundle",
```

Comment:
I cannot believe you are fans of 10/10 🙈

Reply:
cuz I'm well educated! must play and you don't want to miss!